spec/features/consumer/shopping/cart_spec.rb

```diff
@@ -56,9 +56,14 @@
       let(:variant) { product_tax.variants.first }
 
       before do
+        Spree::Config.set allow_backorders: false
         add_product_to_cart order, product_tax
       end
 
+      after do
+        Spree::Config.set allow_backorders: true
```

Comment:
keep in mind that stuff like `Spree::Config.set()` is cached by Rails and affects other tests 💥